src/components/add-review-form/add-review-form.tsx

```diff
@@ -1,12 +1,21 @@
-import React, {Fragment} from "react";
-import PropTypes from "prop-types";
+import * as React from "react";
 
 const NO_RATING = 0;
 const REVIEW_LENGTH_MIN = 50;
 const REVIEW_LENGTH_MAX = 400;
 const starsIds = [1, 2, 3, 4, 5];
 
-const AddReviewForm = (props) => {
+interface Props {
+  children: React.ReactNode | null;
```

Comment:
если проп опциональный, можно указывать в следующем виде

```suggestion
  children?: React.ReactNode;
```